Makefile

```diff
@@ -18,8 +18,12 @@ generate_chain_spec: build
 # Run Docker container in a detached mode
 run-node:
 	@echo "Running the container in detached mode"
-	@bash compile.sh
+	@bash scripts/compile.sh
+ifdef test
```

Comment:
I think that it would be better if we name this property "detached". That way users will know that if they do `make run-node detached` it will run as detached

Reply:
cool